docs/.vuepress/config.js

```diff
@@ -1,26 +1,26 @@
 module.exports = {
   theme: 'cosmos',
-  title: 'Tendermint Core',
+  title: 'CometBFT',
   base: process.env.VUEPRESS_BASE,
   themeConfig: {
-    repo: 'tendermint/tendermint',
-    docsRepo: 'tendermint/tendermint',
+    repo: 'cometbft/cometbft',
+    docsRepo: 'cometbft/cometbft',
     docsDir: 'docs',
     editLinks: true,
     label: 'core',
     algolia: {
       id: "QQFROLBNZC",
       key: "f1b68b96fb31d8aa4a54412c44917a26",
-      index: "tendermint"
+      index: "cometbft"
     },
     versions: [
       {
         "label": "v0.34 (latest)",
         "key": "v0.34"
       },
       {
-        "label": "v0.33",
-        "key": "v0.33"
+        "label": "v0.37",
```

Comment:
Is this supposed to be the last version or the next version?

Reply:
The previous, right?

Reply:
Resolving because this (Vuepress stuff) is going be remove, in any case.